src/controllers/initgamecontroller.js

```diff
@@ -93,6 +93,31 @@ class InitGameController extends Controller
             $(this).css({opacity: 1});
         });
 
+        $("#help-factory").click(function () {
```

Comment:
This whole long list can be refactored into a single generic function that takes `anchor, fn` as arguments. If you have time, else I'll probably do so somewhere in the coming weeks.